mirrord/cli/src/config.rs

```diff
@@ -264,6 +264,9 @@ pub(super) struct ExtensionExecArgs {
     /// Specify target
     #[arg(short = 't')]
     pub target: Option<String>,
+    /// User executable - the executable the layer is going to be injected to.
+    #[arg(short = 'e')]
```

Comment:
Do we want to expose this to users? If not maybe use an env var instead?

Reply:
This is an argument to the extension command, so none of this should be exposed to the users. So, as far as I can see this argument is not different from the other arguments in this config in this regard.